src/nycdb/dof_parser.py

```diff
@@ -1,61 +0,0 @@
-"""
-DOF rolling sales data is downloadable only as an excel document.
-This file process that file so that we can treat like we do csvs.
-"""
-import xlrd
```

Comment:
Are there any modules left that use `xlrd`?  If this was the last one, we might be able to remove it as a dependency from nycdb!

Reply:
If it is the last place (and it looks like it is) you should remove this line too:

https://github.com/nycdb/nycdb/blob/6dc3442b729b49042ee11bb31fe997f0fbb160be/src/setup.py#L37